lib/logstash/outputs/http.rb

```diff
@@ -8,18 +8,18 @@
 
 class LogStash::Outputs::Http < LogStash::Outputs::Base
   include LogStash::PluginMixins::HttpClient
-  
+
   concurrency :shared
 
   attr_accessor :is_batch
 
   VALID_METHODS = ["put", "post", "patch", "delete", "get", "head"]
-  
+
   RETRYABLE_MANTICORE_EXCEPTIONS = [
     ::Manticore::Timeout,
     ::Manticore::SocketException,
-    ::Manticore::ClientProtocolException, 
-    ::Manticore::ResolutionFailure, 
+    ::Manticore::ClientProtocolException,
+    ::Manticore::ResolutionFailure,
```

Comment:
there are still these unrelated (to the testing changes) Manticore changes, is that an oversight?

Reply:
it was just some whitespace cleanup, I can leave it out of the PR if you want

Reply:
oh sorry, my bad - I saw the other WS cleanups but for some reason I thought these were additions. NVMD.